faucet/valve_flood.py

```diff
@@ -408,12 +411,12 @@ def _reset_peer_distances(self):
 
     def _build_flood_rule_actions(self, vlan, exclude_unicast, in_port,
                                   exclude_all_external=False, exclude_restricted_bcast_arpnd=False):
-        exclude_ports = []
+        exclude_ports = self._inactive_away_stack_ports()
         external_ports = vlan.loop_protect_external_ports()
 
         if in_port and in_port in self.stack_ports:
             in_port_peer_dp = in_port.stack['dp']
-            exclude_ports = [
+            exclude_ports = exclude_ports + [
```

Comment:
Why change all_peer_ports to a set but not exclude_ports

Reply:
all_ports was already a set (the python notation starting with curly-brace), it's just a different (more succinct) way of specifying it (and makes lint more happy). I don't think it mattered if exclude_ports was a set or not (just depends on how it's used), so I didn't muck with it if I didn't need to!